.github/contributing.md

```diff
@@ -50,15 +56,20 @@ When running the service with this command, it will be configured to run using t
 
 Enter the web app container (`app`). The shell prompt changes indicating subsequent commands will run in the container:
 
-```
-root@<container ID>:/app/woeip# make local.shell
+```bash
+make local.shell
 ```
 
 Create a Django superuser:
 
 ```
 root@<container ID>:/app/woeip#	./manage.py createsuperuser 
 ```
+Enter a username, password and email. These are for logging into the docker container. 
```

Comment:
The superuser is for the Django Admin

Reply:
will add

Reply:
I committed the change directly to your branch; hope that's okay! I figured it was a minor enough change.